sale_order_invoicing_queued/models/sale_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+try:
+    from odoo.addons.queue_job.job import job
+except ImportError:
+    import logging
+    import functools
+
+    logging.getLogger().debug("Can't `import queue_job`.")
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
```

Comment:
```suggestion
```

Reply:
Not sure this change will work

Reply:
Yes, it will :wink: (if you do https://github.com/OCA/account-invoicing/pull/588#discussion_r319387683 too).